pom.xml

```diff
@@ -16,7 +16,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <grpc.version>1.33.1</grpc.version>
     <protobuf.version>3.13.0</protobuf.version>
-    <dapr.proto.baseurl>https://raw.githubusercontent.com/dapr/dapr/f3ddd345962d7e065e26a54f7b2f1f2db9af1fdb/dapr/proto</dapr.proto.baseurl>
+    <dapr.proto.baseurl>https://raw.githubusercontent.com/artursouza/dapr/e582d2b6bf6305ed7bd99a784002e04ee52508b6/dapr/proto</dapr.proto.baseurl>
```

Comment:
I will update this to dapr/dapr repo once my other PR is merged. I need that for proto and ITs to pass:

https://github.com/dapr/dapr/pull/2477

Reply:
Updated.